airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -180,7 +180,7 @@ def validate_form(self, data, **kwargs):
 class SetSingleTaskInstanceStateFormSchema(Schema):
     """Schema for handling the request of updating state of a single task instance."""
 
-    dry_run = fields.Boolean(dump_default=True)
+    dry_run = fields.Boolean(dump_default=True, load_default=True)
```

Comment:
Nit: if it works without `dump_default=True`, I prefer to remove it, it seems useless
```suggestion
    dry_run = fields.Boolean(load_default=True)
```